setup.py

```diff
@@ -65,6 +65,8 @@
         'Bottleneck==1.3.2',
         'python-json-logger',
         'rlguard-lib',
+        'openeo_udf',
```

Comment:
Is this dependency needed? I don't see an import of something from this package.